src/app/shared/custom-components/toolbar/search/search.component.html

```diff
@@ -1,11 +1,13 @@
-<form class="example-form">
+<form class="search-form">
   <input
     type="text"
-    class="search"
+    class="search-form__search"
     placeholder="Search"
     [formControl]="control"
     [matAutocomplete]="auto"
   />
+  <button class="search-form__search-button-image"></button>
```

Comment:
I think you can use only ```search-form__button```, since you need to specify one element (now there are two of them - picture and button)